### ISSUE TITLE
🐞 @microsoft/generator-sharepoint package.json misses lib entry

### ISSUE THREAD

**Issue by StfBauer** [SharePoint/sp-dev-docs#5070]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
@microsoft/generator-SharePoint package.json should contain a lib folder in order to make path.resolve working.

#### Observed Behavior

In a project that has @microsoft/generator-sharepoint as a dependency installed. ```path.resolve``` does not work.

```json
{
  "name": "pathresel",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "gulp": "^4.0.2"
  },
  "dependencies": {
    "@angular/cli": "^8.3.21",
    "@microsoft/generator-sharepoint": "^1.9.1"
  }
}
```

The following script will fail because of the missing lib entry.

```
const path = require('path');
const msgen = require.resolve('@microsoft/generator-sharepoint');

console.log(msgen);
```

<img width="641" alt="Screenshot 2019-12-31 at 16 12 37" src="https://user-images.githubusercontent.com/5503835/71625489-654a5a00-2be8-11ea-98ad-7f27cbee14e4.png">

It causes issues in the PnP/generator using any other package manager and unit test other than npm.

#### Steps to Reproduce
1. Create new npm package
2. Install @microsoft/generator-sharepoint as a dependency
3. Run the script above

**Comment by KevinTCoughlin** [comment]
Thanks @StfBauer, I've committed a fix. This will be resolved next drop. cc: @patmill @VesaJuvonen

**Comment by KevinTCoughlin** [comment]
This was fixed in `@microsoft/generator-sharepoint@1.11`, see https://unpkg.com/browse/@microsoft/generator-sharepoint@1.11.0/package.json/. I believe this issue can be closed.
